cost/aws/instance_cloudwatch_utilization/aws_instance_cloudwatch_utilization.pt

```diff
@@ -0,0 +1,666 @@
+name "AWS Instance CloudWatch Utilization"
+rs_pt_ver 20180301
+type "policy"
+short_description "Checks AWS Regions configured in RightScale and gets CloudWatch data for instances. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/native_utilization/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
Should this be "AWS Instances"? We should also include a note about the timeframe calculated.
Example: "Reports on AWS instances CloudWatch Utilization over the past 30 days..."